<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name = "styleSheet">
            <string notr="true">
                background-color: white;
                color: black;
            </string>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>852</width>
                <height>514</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Task Planner</string>
        </property>
        <widget class="QCalendarWidget" name="calendarWidget">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>150</y>
                    <width>411</width>
                    <height>311</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">font:12pt;
                    background-color: white;
                    color:black;
                </string>
            </property>
        </widget>
        <widget class="QListWidget" name="tasksListWidget">
            <property name="geometry">
                <rect>
                    <x>480</x>
                    <y>180</y>
                    <width>341</width>
                    <height>271</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">font:13pt;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="deleteButton">
            <property name="geometry">
                <rect>
                    <x>480</x>
                    <y>460</y>
                    <width>341</width>
                    <height>28</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">border-radius:10px;
                    background-color: #70161E;
                    color:white;
                    font:12pt;</string>
            </property>
            <property name="text">
                <string>Delete Task</string>
            </property>
        </widget>
        <widget class="QPushButton" name="addButton">
            <property name="geometry">
                <rect>
                    <x>730</x>
                    <y>110</y>
                    <width>93</width>
                    <height>28</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">border-radius:12px;
                    background-color: #70161E;
                    color:white;
                </string>
            </property>
            <property name="text">
                <string>Add new</string>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>852</width>
                    <height>101</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">font-size : 24pt;
                    background : #70161E;
                    color: white;
                </string>
            </property>
            <property name="text">
                <string>Task Planner</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="taskLineEdit">
            <property name="geometry">
                <rect>
                    <x>480</x>
                    <y>110</y>
                    <width>241</width>
                    <height>31</height>
                </rect>
            </property> 
            <property name="styleSheet">
                <string notr="true">font:12pt;</string>
            </property>
        </widget>
        <widget class="QTimeEdit" name = "timeBox">
            <property name="geometry">
                <rect>
                    <x>480</x>
                    <y>145</y>
                    <width>120</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">font:14pt;</string>
            </property>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>